os/arch/arm/src/armv7-m/mpu.h

```diff
@@ -160,6 +160,12 @@
 #define MPU_RASR_AP_RORO      (6 << MPU_RASR_AP_SHIFT)	/* P:RO   U:RO   */
 #define MPU_RASR_XN             (1 << 28)	/* Bit 28: Instruction access disable */
 
+#ifdef CONFIG_MPU_STACK_OVERFLOW_PROTECTION
+#define STACK_GUARD_ADDR_MASK	0xffffffe0	/* Bits N-31:  Used for MPU Region base addrese */
+#define MPU_REGION_ADDRESS_ALIGNMENT 32	/* MPU regions must be 32 byte address aligned */
+#define MPU_REGION_SIZE_ALIGNMENT 32		/* Min MPU region size is 32 bytes */
```

Comment:
In this define, we dont need to mention ALIGNMENT, because, this define indicates the size of MPU stack guard and not its alignment.